README.md

```diff
@@ -2,11 +2,41 @@
 
 [![CircleCI](https://circleci.com/gh/pkamenarsky/concur-replica.svg?style=svg)](https://circleci.com/gh/pkamenarsky/concur-replica)
 
-`concur-replica` is a [Concur](https://github.com/ajnsit/concur) backend for [Replica](https://github.com/pkamenarsky/replica). As such, you might want to make yourself familiar with both projects first.
+Build interactive web UIs using nothing but Haskell.
+
+No user-written JS. No transpiler.
+
+## How?!
+
+[Replica](https://github.com/pkamenarsky/replica): a remote virtual DOM library. Clients run a fixed JavaScript snippet which opens a connection to your Haskell server. The client uses it to notify the server of browser events, and the server uses it to send DOM updates to be applied by the client.
+
+[Concur](https://github.com/ajnsit/concur): a UI framework that generalizes the Elm architecture. It's based around `Widget`s, sections of the UI that may return a value.
+
+## Upsides and downsides
+
+This strategy is well understood due to its use by [Phoenix LiveView](https://hexdocs.pm/phoenix_live_view) (among other projects).
+
+There are two situations where you should rule `concur-replica` out completely:
+
++ Instant feedback is required
+
++ It needs to work offline
+
+The strategy also has two softer costs:
+
++ Animations costs bandwidth as well as CPU
```

Comment:
Probably good to qualify as well - css animations are more or less "zero cost" (e.g. update animated css class at start and end of animation). However, continuously updating attributes 20 times a second is not a good idea.

Reply:
How would you like to phrase it? "Non-CSS animations" / "animations done in the general purpose language"?

Reply:
"Animations costs bandwidth as well as CPU if not done using CSS" ?

Reply:
I'd say that ties into the previous point - i.e. everything that requires a stream of events upstream (Google Maps) or downstream (Google Maps, custom animations) is out. I think if we go with "No client side logic is required" or something similar that'd cover the animations as a sub-case as well.